docs/formats.rst

```diff
@@ -234,6 +234,20 @@ The ``import_set()`` method also supports a ``skip_lines`` parameter that you
 can set to a number of lines that should be skipped before starting to read
 data.
 
+The ``export_set()`` method supports a ``column_width`` parameter. Depending on the
+value you pass, the column width will be set accordingly. It can be either None, int, or "adaptive".
+If "adaptive" is passed, the column width will be unique for every column and will be 
+calculated based on values' length. Example of usage
+
+```python3
+data = tablib.Dataset()
+data.export('xlsx', column_width='adaptive')
+```
```

Comment:
This is RST not Markdown, so we can use a double colon at the end of the last line to activate the code block:

```suggestion
calculated based on values' length. For example::

    data = tablib.Dataset()
    data.export("xlsx", column_width="adaptive")